bindings/python/src/model/representation/core/section.h

```diff
@@ -102,38 +96,35 @@ namespace geode
                     return components;
                 },
                 pybind11::return_value_policy::reference )
-            .def( "nb_embedded_corner_surfaces",
```

Comment:
what about embedded corner inside a line? maybe not different from an internal one